src/use/router/index.ts

```diff
@@ -122,6 +123,15 @@ const router = createRouter({
                 roles: ["Lecturer", "Admin"],
             },
         },
+        {
+            path: "/createExamReg",
+            name: "examRegForm",
+            component: ExamRegForm,
+            meta: {
+                title: "Examination Registration Creation" + suffix,
```

Comment:
```suggestion
                title: "Examination Regulation Creation" + suffix,
```
Note: The title is used for displaying something as the tab title in the browser. "Examination Regulation Creation" could be a bit long, we could use the short form "Exam" like in the navbar.